Simulations/python/makeCalibPrototypes.py

```diff
@@ -430,18 +430,52 @@ def generateStaticCalibs(outputDir):
     primaryhdu = fits.PrimaryHDU()
     primaryhdu.header['HIERARCH ESO PRO CATG'] = "PERSISTENCE_MAP"
     primaryhdu.header['INSTRUME'] = "METIS"
-    primaryhdu.header['HIERARCH ESO DPR TECH'] = "IMAGE,N"
-    primaryhdu.header['HIERARCH ESO DPR TYPE'] = "PERSISTENCE"
-    primaryhdu.header['HIERARCH ESO DPR CATG'] = "CALIB"
-    primaryhdu.header['HIERARCH ESO INS MODE'] = "img_n"
+    primaryhdu.header['HIERARCH ESO PRO TECH'] = "IMAGE,LM"
+    primaryhdu.header['HIERARCH ESO PRO TYPE'] = "PERSISTENCE"
+    primaryhdu.header['HIERARCH ESO INS MODE'] = "img_lm"
     primaryhdu.header['HIERARCH ESO DRS FILTER'] = "OPEN"
     primaryhdu.header['HIERARCH ESO DRS NDFILTER'] = "OPEN"
 
     data = np.zeros((2048,2048))
     hdu = fits.ImageHDU(data, name="PERSISTENCE_MAP")
     hdul = fits.HDUList([primaryhdu,hdu])
+    hdul.writeto(f"{outputDir}/PERSISTENCE_MAP_LM.fits",overwrite=True)
+
+    primaryhdu = fits.PrimaryHDU()
+    primaryhdu.header['HIERARCH ESO PRO CATG'] = "PERSISTENCE_MAP"
+    primaryhdu.header['INSTRUME'] = "METIS"
+    primaryhdu.header['HIERARCH ESO PRO TECH'] = "IMAGE,N"
+    primaryhdu.header['HIERARCH ESO PRO TYPE'] = "PERSISTENCE"
+    primaryhdu.header['HIERARCH ESO INS MODE'] = "img_n"
+    primaryhdu.header['HIERARCH ESO DRS FILTER'] = "OPEN"
+    primaryhdu.header['HIERARCH ESO DRS NDFILTER'] = "OPEN"
```

Comment:
These two, DRS.FILTER and DRS.NDFILTER do not really make sense for a persistence map. I suggest to just remove them.
```suggestion
```